.github/workflows/release.yml

```diff
@@ -32,12 +33,15 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: Publish to npm
-        run: yarn release
+        uses: changesets/action@v1
+        with:
+          publish: yarn release
+          version: yarn stage
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.SEEK_OSS_CI_NPM_TOKEN }}
 
       - name: Publish Storybook
         run: yarn storybook:build && yarn storybook:deploy -- --ci --host-token-env-variable=GITHUB_TOKEN
         env:
-          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
```

Comment:
I'm guessing we may need to remove `github.actor` if we want to make this change, as the org token belongs to https://github.com/orgs/seek-oss/teams/ci.

That said, can we check if we can stick to the repo-scoped GITHUB_TOKEN for the Storybook step? (I can't quite remember at this time why it's insufficient for Changesets though!)

Reply:
I suppose if it already works it should be fine?